### ISSUE TITLE
[Transfer to DST] a11y Discovery: Breadcrumb Link Cursor Event

### ISSUE THREAD

**Issue by coforma-terry** [department-of-veterans-affairs/va.gov-team#53331]
## Transfer Ticket
`Needs reassigned to the new DST specialist`

## a11y CAIA Asignee(s)
- @SarahKay8 

## Issue Description
Should a breadcrumb link have a cursor event to maintain consistency?
Related to ticket https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1458

## Tasks
Discovery - do we need a cursor event or not?

## Acceptance Criteria
Have Josh review findings and then determine if this needs to be implemented

**Comment by SarahKay8** [comment]
Now:

Here’s the desk research we’ve conducted where we’ve done searches in a11yslack, a11y trusted resources, and talked to the content team
Based off of it, we’re tentatively deciding to go with making the current page a `link`
We can’t be 100% sure this is the right choice as it’s based off more on anecdotes than actual testing
Next:
This issue can be broken down into 2 parts:
Major issue of having a component that is both a link and not a link at once
Minor issue of knowing if we’ve made the right choice between it being a link or not a link
To address the minor issue, understanding the only way to truly answer it is to conduct user testing, we should begin groundwork on hiring a DST a11y specialist
Later:
when said person is hired, we should create an a11y strategy for DST to help facilitate these choices.

Placing this in the backlog as we wait for a new hire for DST. 
For reference- ![Screenshot 2023-02-14 at 1.28.33 PM.png](https://images.zenhubusercontent.com/623cd4d6774110eeb19922cb/6678db0b-9eb3-4420-b688-f7f958337884)![Screenshot 2023-02-14 at 1.28.43 PM.png](https://images.zenhubusercontent.com/623cd4d6774110eeb19922cb/c40cd825-4bf5-4fe0-9aec-37895d4747a6) @coforma-terry @RLHecht

**Comment by mnorthuis** [comment]
@coforma-terry Just dropping a note to establish that any changes to navigational components on the site should have IA involvement in discussion and design.  Thanks!

**Comment by coforma-terry** [comment]
Thanks @mnorthuis - added Jenny and Kristin for visibility.

**Comment by sara-amanda** [comment]
## CAIA Transfer
@SarahKay8 would we transfer this ticket to the new DST a11y specialist?

**Comment by SarahKay8** [comment]
@sara-amanda yes that would be great!

**Comment by kristinoletmuskat** [comment]
Popping in to say that we'd just like to be kept abreast of changes to the breadcrumbs, but we're totally down to follow a11ys lead on this! If the new DST person wants to loop us in when they've done their discovery that would be great.

**Comment by sara-amanda** [comment]
Moving to `validate` status on Sitewide CAIA pipeline, so we can transfer to @rsmithadhoc
cc: @coforma-terry

**Comment by rsmithadhoc** [comment]
Re-assigning to @micahchiang for initial review.

**Comment by micahchiang** [comment]
@sara-amanda @coforma-terry @SarahKay8 - Hey there, I'm the engineering lead for the Design System Team. My understanding of this issue, based on the task listed in the description, is that it is still a discovery ticket that needs some consensus from multiple parties, not necessarily implementation or updates to a Design System component, just yet. 

With that, I am reassigning to @SarahKay8 and @sara-amanda until the work is complete on reaching a11y group consensus on whether or not a cursor event is needed. 

For future reference, the Design System team needs visibility on issues that get submitted to us, so we can properly size, refine, and prioritize them in our backlog. This gets lost when things are directly assigned to a member of our team. Please use the `platform-design-system-team` label on issues, and feel free to drop us a message in the #platform-design-system slack channel. Thanks!

**Comment by SarahKay8** [comment]
After further conversation during a11y standup, article research, feedback from current DST specialist Ryan Smith, no additional user testing is needed. Current page should remain a `link`. [Link to a11y specialist conversation](https://dsva.slack.com/archives/C0335Q88X0S/p1692897954788099). Article references -

**Comment by laflannery** [comment]
@SarahKay8 I was just reading this and following up because it relates to something on my team and I want to ask a couple clarifying questions:
- The final decision was for the current page item within a pager to be a link but does this means:
  - Should this current menu item link be actionable?
  - If a mouse user hovers over the element, should the cursor change to a hand?
  - Would a keyboard user be able to tab to the current menu item?

**Comment by SarahKay8** [comment]
@laflannery 1. Should this current menu item link be actionable?

Because the other breadcrumb items are actionable links, making the current page link actionable maintains consistency in the user interface. Also users might expect the current page to be clickable for navigational purposes, especially if other breadcrumbs are clickable.

2. If a mouse user hovers over the element, should the cursor change to a hand? 

yeah if the current menu item is a link, it's good practice to change the cursor to a hand on hover to indicate interactive nature. This helps convey to users that the element is clickable.

3. Would a keyboard user be able to tab to the current menu item?

If the current menu item is a link, it should be included in the tab order

**Comment by mnorthuis** [comment]
@SarahKay8 I cannot access the slack conversation linked above and would like to see the background and discussion.  What channel is the conversation in?

**Comment by SarahKay8** [comment]
Just realizing this is in our #a11y-specialists channel which is private. However, here are some screenshots from the convo @mnorthuis 

![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/102324990/6c84ebcb-c0ac-4774-83ae-9560b7aa6add)
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/102324990/373d5fc3-1252-4c7e-ac8f-406d8b97dc43)
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/102324990/b9a8f117-fcd0-4502-882b-b2e068839976)
